### PR TITLE
Update code-of-conduct.md

### PR DESCRIPTION
**Pull request by spiffxp**
Refer to kubernetes/community as authoritative source for code of conduct

ref: kubernetes/community#1527

### CODE REVIEW

File: code-of-conduct.md [hunk 1]
```diff
@@ -1,55 +1,3 @@
-## Kubernetes Community Code of Conduct
+# Kubernetes Community Code of Conduct
```
Comment (TomSweeneyRedHat) [inline code-of-conduct.md]: Should this be "Kubernetes" or "CRI-O"?

**Comment by spiffxp** [comment]
I'm operating under the impression that any  repo under kubernetes-incubator is part of the kubernetes project.  (ref: https://github.com/kubernetes/community/issues/1407)

I would prefer that all repos use the exact same code-of-conduct.md file rather than slightly different variants.  The reason I'd rather we not link directly to the CNCF code of conduct is to prevent further all-repo-updates in the event that we change the code of conduct by which Kubernetes operates, even if it's something as trivial as s/CNCF/Kubernetes.  (ref: https://github.com/kubernetes/community/issues/1527)

File: code-of-conduct.md [hunk 2]
```diff
@@ -1,55 +1,3 @@
...
-While at Kubernetes events or related social networking opportunities, attendees should not engage in
-discriminatory or offensive speech or actions regarding gender, sexuality, race, or religion.  Speakers should
-be especially aware of these concerns.
-
-The Kubernetes team does not condone any statements by speakers contrary to these standards.  The Kubernetes
-team reserves the right to deny entrance and/or eject from an event (without refund) any individual found to
-be engaging in discriminatory or offensive speech or actions.
-
-Please bring any concerns to the immediate attention of the Kubernetes event staff.
+Please refer to our [Kubernetes Community Code of Conduct](https://git.k8s.io/community/code-of-conduct.md)
```
[the remark this answers is not in the document]
Comment (runcom) [reply code-of-conduct.md]: I'm ok linking you kube

**Comment by runcom** [comment]
LGTM

**Comment by runcom** [comment]
Also, please sign your commit, Travis is failing the validation check

**Pull request merged**